### ISSUE TITLE
Hover tips should be accessible for keyboard and screen reader users

### ISSUE THREAD

**Issue by swongCO** [flexion/ef-cms#10548]
As a keyboard-only and screen reader user, so that I can access information, I need hover tips to be accessible.

### Description
Hover tips/pop-ups are not accessible for keyboard only and screen readers. Most hover tips are on icons but there are others (like default browser pop-ups) that are not accessible.

### Acceptance Criteria
* Hover tips are accessible via keyboard only and screen readers
* VoiceOver does not read pop-up browser default error messages: Firefox in Log in screen
* Determine if user documentation needs to be updated (Tenille): Y / N
